caddy/install.ps1

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env pwsh
+
+#################
+# Install caddy #
+#################
+
+# Every package should define these variables
+$pkg_cmd_name = "caddy"
+
+$pkg_dst_cmd = "$Env:USERPROFILE\.local\bin\caddy.exe"
+$pkg_dst = "$pkg_dst_cmd"
+
+$pkg_src_cmd = "$Env:USERPROFILE\.local\opt\caddy-v$Env:WEBI_VERSION\bin\caddy.exe"
+$pkg_src_bin = "$Env:USERPROFILE\.local\opt\caddy-v$Env:WEBI_VERSION\bin"
+$pkg_src_dir = "$Env:USERPROFILE\.local\opt\caddy-v$Env:WEBI_VERSION"
+$pkg_src = "$pkg_src_cmd"
+
+$pkg_download = "$Env:USERPROFILE\Downloads\$Env:WEBI_PKG_FILE"
+
+# Fetch archive
+IF (!(Test-Path -Path "$Env:USERPROFILE\Downloads\$Env:WEBI_PKG_FILE")) {
+    # TODO: arch detection
+    echo "Downloading caddy from $Env:WEBI_PKG_URL to $pkg_download"
+    & curl.exe -A "$Env:WEBI_UA" -fsSL "$Env:WEBI_PKG_URL" -o "$pkg_download.part"
+    & move "$pkg_download.part" "$pkg_download"
+}
+
+IF (!(Test-Path -Path "$pkg_src_cmd")) {
+    echo "Installing caddy"
+
+    # TODO: create package-specific temp directory
+    # Enter tmp
+    pushd .local\tmp
+
+    # Remove any leftover tmp cruft 
+    Remove-Item -Path ".\caddy-v*" -Recurse -ErrorAction Ignore
+    Remove-Item -Path ".\caddy.exe" -Recurse -ErrorAction Ignore
+
+    # Unpack archive file into this temporary directory
+    # Windows BSD-tar handles zip. Imagine that.
+    echo "Unpacking $pkg_download"
+    & tar xf "$pkg_download"
+    & dir
+
+    # Settle unpacked archive into place
+    echo "Install Location: $pkg_src_cmd"
+    New-Item "$pkg_src_bin" -ItemType Directory
```

Comment:
Did you test this on Windows? I thought that the Caddy releases didn't have subdirectories? Am I mistaken?

To test you should be able to run this in cmd.exe:

```cmd
node _webi/test.js caddy/
powershell -ExecutionPolicy Bypass install-caddy.ps1
```

Reply:
Did you perhaps delete it? You may need to run `node _webi/test.js caddy/` again.